tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -163,7 +178,8 @@ RUN dnf -y install \
     unzip && \
     npm install -g n && n 16.13.1 && npm install -g npm@8.5.0 && dnf remove -y nodejs
 
-RUN pip3 install black git+https://github.com/coderanger/supervisor-stdout setuptools-scm
+RUN pip3 install -vv black git+https://github.com/coderanger/supervisor-stdout setuptools-scm
```

Comment:
this was already here, but it's gross. We shouldn't be pulling in git repos  without shas...
You don't need to do anything about this now, but it does break ones heart.

Reply:
done